lib/components/rtsp-session/index.ts

```diff
@@ -330,8 +354,11 @@ export class RtspSession extends Tube {
     this.n0 = {}
     this.t0 = {}
     this.clockrates = {}
+
+    const sessionControlURL = this._controlURL(msg.sdp.session.control)
```

Comment:
Is it necessary to have this variable?
Couldn't the value be assigned directly to `this._sessionControlURL` instead?

Reply:
Seems like you are correct. `this._sessionControlURL` is never reassigned anywhere else, so no point in using a local copy.

Reply:
It was a simple fix to make sure TypeScript knows it's not undefined or null later on in the `.forEach` callbacks.